inaugurator/partitiontable.py

```diff
@@ -9,8 +9,9 @@ class PartitionTable:
     _DEFAULT_SIZES_GB = dict(
         smallSwap=1,
         bigSwap=8,
-        minimumRoot=14,
-        createRoot=30)
+        minimumRoot=12,
+        maximumRoot=18,
+        createRoot=15)
```

Comment:
If you touching this part - why not to parametrize it?
Anyhow, not sure I would go for it before we understand how it affects the inaugurations - how much storage space the system consumes once you move everything to data partition?
Another option is to create another cache partition for the inauguration cache.